shell/imports/server/migrations.js

```diff
@@ -787,6 +787,49 @@ function setIpBlacklist(db, backend) {
   }
 }
 
+function notifyIdentityChanges(db, backend) {
+  // Notify users who might be affected by the identity model changes.
+  //
+  // Two types of users are affected:
+  // - Users who have multiple identities with differing names.
+  // - Users who have identities that are shared with other users.
+  //
+  // However, the second group seems like it must be a subset of the first group. So we only check
+  // for the first.
```

Comment:
Also there are probably like two people in the world that actually have the setup you describe and they are both in this thread. :)